recipe/meta.yaml

```diff
@@ -31,17 +26,16 @@ requirements:
     - pkg-config
     - gperf
     - expat
-    - gettext                     # [not win]
+    - gettext     # [not win]
     - libtool
     - automake
     - autoconf
-    - patch       # [not win]
-    - m2-patch    # [win]
- 
+
   host:
     - freetype
     - libxml2
-    - libuuid                     # [linux]
+    - unzip
+    - coreutils  # [osx]
```

Comment:
Is this for gtouch? If so, please make sure to remove it if we remove the alias.

Reply:
`coreutils` was added for `gtouch`.